.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres:11.6-alpine
+        image: postgis/postgis:12-3.1-alpine
```

Comment:
production has 12.8, not that it's that relevant

Reply:
I think the `3.1` here is the PostGIS version, only the `12` refers to Postgres itself